### PR TITLE
Make compatible with nikic/php-parser:3.0

### PR DESCRIPTION
**Pull request by PavelJurasek**
Hi,
I would like to update compatibility of this tool with nikic/php-parser:3.0.
This unfortunately breaks ```list.test``` and I'm not really sure how to fix it. 
Thanks for suggestions

### CODE REVIEW

**Comment by nikic** [comment]
Could you please change the indentation to 4 spaces to stay consistent?

File: lib/PHPCfg/Parser.php [hunk 1]
```diff
@@ -1010,9 +1014,16 @@ protected function parseExpr_Isset(Expr\Isset_ $expr) {
         );
     }
 
+	protected function parseExpr_List(Expr\List_ $expr) {
```
Comment (nikic) [inline lib/PHPCfg/Parser.php]: This should not be necessary, as list() assignments are handled by parseListAssignment().

File: lib/PHPCfg/Parser.php [hunk 2]
```diff
@@ -1010,9 +1014,16 @@ protected function parseExpr_Isset(Expr\Isset_ $expr) {
...
+		return new Op\Expr\List_(
+			$this->parseExprList($expr->items),
+			$this->mapAttributes($expr)
+		);
+	}
+
     protected function parseListAssignment(Expr\List_ $expr, Operand $rhs) {
         $attributes = $this->mapAttributes($expr);
-        foreach ($expr->vars as $i => $var) {
+        foreach ($expr->items as $i => $var) {
```
Comment (nikic) [inline lib/PHPCfg/Parser.php]: `$var` is now either null or an ArrayItem, the variable needs to be taken from that ArrayItem first. This is probably the reason why you ran into the undefined methods.

**Comment by PavelJurasek** [comment]
Thank you for comments. Tests are passing now.

**Comment by nikic** [comment]
Merged as https://github.com/ircmaxell/php-cfg/commit/10938301e56154daf24c35ef6c4f9a235f8610be with some more updates in https://github.com/ircmaxell/php-cfg/commit/1cc6c31fff4356b8e601335b4497f17366eb6a07. Thanks!

**Pull request closed without merging**